lib/plugins/aws/deploy/lib/cleanupS3Bucket.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
       })
       .then(response => {
         const stacks = findAndGroupDeployments(response, prefix, service, stage);
-        const stacksToKeep = _.takeRight(stacks, stacksToKeepCount);
+        const stacksToKeep = stacks.slice(stacksToKeepCount ? -stacksToKeepCount : stacks.length);
```

Comment:
I think it can be just `stacks.slice(-stacksToKeepCount)`. Do I miss something?

Reply:
@medikoo when `stacksToKeepCount` is count is `0` ,`stacks.slice(-stacksToKeepCount)` will return the original array.

Reply:
Thanks! Indeed. In that case let's maybe put it as `stacks.slice(-stacksToKeepCount || Infinity)`

Reply:
Yes, that will be better. Thanks!

Reply:
Updated PR with the suggestion from @medikoo